### PR TITLE
[CoqIDE] Do not scroll to start of input in navigation commands.

### LINKED ISSUE #15230: Resetting coq moves scroll in coqide
#### Description of the problem

When pressing "Reset Coq" the window scrolls all the way up to the top. This is different to past behaviour where it wouldn't scroll the window. The cursor is still where it was so the scrolling seems completely unnecessary.

#### Coq Version
8.15

### PR DESCRIPTION
**Pull request by ppedrot**
This restores the behaviour from before #14644.

Fixes #15230.

### CODE REVIEW

**Comment by jfehrle** [comment]
Give me a chance to test this before merging.

**Review by Alizter — CHANGES_REQUESTED** [review]
LGTM but let's wait for @jfehrle's feedback.

File: ide/coqide/coqOps.ml
Comment (jfehrle) [inline ide/coqide/coqOps.ml:907]: You like formatting the statement this way?

**Comment by jfehrle** [comment]
It looks good to me, thanks.

File: ide/coqide/coqOps.ml
Comment (ppedrot) [reply ide/coqide/coqOps.ml:907]: I reverted to the previous state for diff cleanliness purposes. Otherwise I do tend to prefer having `then` on the same line as `if` indeed.

**Review by Alizter — APPROVED** [review]

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**
